styles.css

```diff
@@ -147,37 +160,35 @@ fieldset input {
   padding: 5px 0 30px 0;
 }
 
-
 .guess-box {
   width: 50%;
   text-align: center;
 }
 
-.current-guess {
+.card_results .flex-parent .current-guess {
   height: 100px;
 }
 
-.left-guess {
-  border-right: 2px solid #cecece;
+.card_results .flex-parent .guess-box:last-of-type {
+  border-left: 2px solid #cecece;
```

Comment:
Oh, I didn't know that you could use pseudo-elements in that way. That seems like a good way for us to keep our class names to a minimum.

Reply:
Yeah, pseudo elements can be super useful!